app/controllers/authenticate_controller.rb

```diff
@@ -128,23 +128,24 @@ def k8s_inject_client_cert
   private
 
   def handle_login_error(err)
-    logger.debug("Login Error: #{err.inspect}")
+    logger.info("Login Error: #{err.inspect}")
     err.backtrace.each do |line|
       logger.debug(line)
     end
 
     case err
     when Errors::Authentication::Security::AuthenticatorNotWhitelisted,
       Errors::Authentication::Security::WebserviceNotFound,
-      Errors::Authentication::Security::AccountNotDefined
+      Errors::Authentication::Security::AccountNotDefined,
+      Errors::Authentication::Security::RoleNotFound
       raise Unauthorized
     else
       raise err
     end
   end
 
   def handle_authentication_error(err)
-    logger.debug("Authentication Error: #{err.inspect}")
+    logger.info("Authentication Error: #{err.inspect}")
     err.backtrace.each do |line|
       logger.debug(line)
```

Comment:
I see this is repeating itself, shouldn't we refactor this into a method?

Reply:
Done